pkg/util/k8s/unstructured.go

```diff
@@ -67,3 +67,27 @@ func ReplaceObj(objs []*uns.Unstructured, new *uns.Unstructured) []*uns.Unstruct
 
 	return out
 }
+
+// RemoveObjByKindName will remove a given object from the list of objects.
+// It will match on the object's kind, namespace and name. If the object isn't found
+// in the list, it will not be removed.
+func RemoveObjByKindName(objs []*uns.Unstructured, kind, namespace, name string) []*uns.Unstructured {
```

Comment:
This needs to take the Group (e.g.  `apps`) as well, since there are sometimes collisions between groups.

Reply:
oh yea totally forgot that!